ldms/rules.mk

```diff
@@ -0,0 +1,24 @@
+# 1. Fix duplicate titles when converting to man
+# 	- Remove "NAME" and "=====" underneath
+# 	- Add :title: to first sentence after :Date:
+# 2. Remove :ref:`<>` links in .rst to avoid rs2man "Unkown interpretd text role "ref"" errors
+# 	Ex: :ref:`ldms_quickstart(7) <ldms_quickstart>` --> ldms_quickstart(7).
+# 3. Remove generated "/" after every special character.
+# 	Ex: \fB\-a,\fP\fI\-\-default_auth\fP --> \fB-a,\fP\fI--default_auth\fP
+# 4. Remove lines with unnecessary rst2man output.
```

Comment:
Were they causing problems? I'd love to see us able to get rid of all the sed usage.

Reply:
Not sure, I've done some research on this and from what I've found, fst documentation wasn't designed/targeted to be built as man pages.